com.valvesoftware.Steam.Utility.InhibitScreensaver.yml

```diff
@@ -0,0 +1,63 @@
+id: com.valvesoftware.Steam.Utility.InhibitScreensaver
+sdk: org.freedesktop.Sdk//23.08
+runtime: com.valvesoftware.Steam
+runtime-version: stable
+build-extension: true
+appstream-compose: false
+separate-locales: false
+rename-icon: inhibit-screensaver
+rename-desktop-file: inhibit-screensaver.desktop
+command: inhibit-screensaver
+
+finish-args:
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.freedesktop.PowerManagement
```

Comment:
```suggestion
```

This is an extension it won't be used separately from the main app

Reply:
Also maybe use the portal https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.Inhibit.html This will make it work cross desktop and without any permissions

Reply:
I've removed the `finish-args` from the file. Just for clarification: I'm assuming an extension can't specify/get any additional permissions besides the ones inherited from the base package?

Re the inhibit portal: From skimming the docs it looks like I'd need a window handle (either the XID or the handle obtained by the xdg_foreign protocol on Wayland) which I have no idea how to get as the application doesn't talk to or interact with any window management at all.

Reply:
You can just pass an empty as string as Window identifier.

Reply:
I've now implemented use of that portal.
The previous way using the ScreenSaver and PowerManagement APIs is still there and will be used as a fallback solution in case the inhibit portal isn't present.